src/content/app/species/SpeciesPage.tsx

```diff
@@ -35,32 +35,39 @@ import { getPathParameters, useUrlParams } from 'src/shared/hooks/useUrlParams';
 import type { ServerFetch } from 'src/routes/routesConfig';
 import type { AppDispatch } from 'src/store';
 
-const LoadableSpeciesPageContent = loadable(
+const LazylyLoadedSpeciesPageContent = React.lazy(
   () => import('./SpeciesPageContent')
 );
 
+const defaultTitle = 'Species page — Ensembl';
+const defaultDescription = 'Species home page';
```

Comment:
all other places you have pageTitle and pageDescription. Can we keep it the same?